include/openenclave/internal/load.h

```diff
@@ -89,6 +90,9 @@ struct _oe_enclave_image
      * other enclave binary formats are supported later */
     oe_enclave_elf_image_t elf;
 
+    /* Pointer to the image of a module. */
+    oe_enclave_elf_image_t* module;
```

Comment:
Consider a more informative comment and name for `module`. For example
```c
/* Pointer to dependency image for the enclave
 * Only up to one such .so dependency is currently allowed */
oe_enclave_elf_image_t* submodule;
```

Reply:
Done